hrms/www/jobs/index.css

```diff
@@ -0,0 +1,58 @@
+body {
```

Comment:
```suggestion
body.jobs-page {
```
Let's add scoped styles. You can set this class in your context
`context.body_class = "jobs-page"`

<img width="592" alt="image" src="https://github.com/frappe/hrms/assets/24353136/65d7a89a-a09b-4371-b35d-5d36d4a5d905">
 
Same for h3, p, .footer